### DISCUSSION TITLE
Multiple parents

### DISCUSSION THREAD

**Discussion by 1234shel** [bulletphysics/bullet3#4009] [General]
Is it possible for a link to have multiple parents? I am trying to create a square grid of connected cubes but I am realising that this may not be possible if we are restricting the number of parents for each link to one.

**Comment by erwincoumans** [comment]
You can use constraints to close a loop.
One example of using a constraint is here:
https://github.com/bulletphysics/bullet3/blob/master/examples/pybullet/examples/quadruped.py#L276
